src/pages/BookPage.tsx

```diff
@@ -31,17 +48,20 @@ const BookPage: React.FC = () => {
         <NavigationBar backButtonOnClick={() => history.push("/")} />
       </IonHeader>
       <IonItem>
-        <IonSearchbar
+        <IonInput
```

Comment:
OH, when you said you changed it to input I thought you meant HTML input element. Very happy to see we are still able to get this working with an Ion element here. :)

Reply:
Ah yeah at first I did make it `<input>` but then I realized `IonSearchbar != IonInput` so we can stay keep ion here.